src/geode/mesh/core/regular_grid.cpp

```diff
@@ -205,6 +226,20 @@ namespace geode
             return absl::nullopt;
         }
 
+        bool is_vertex_on_border( index_t vertex_id ) const
+        {
+            const auto vertex_indices = vertex_index( vertex_id );

```

Comment:
This line could be optional if we have the second overloads using the Index and already the value of the Index